src/geo/GoogleTiles.ts

```diff
@@ -0,0 +1,64 @@
+import {IGoogleTile, IPoint, IBBox} from "../Geo";
+import {ILatLon} from "../Graph";
+
+export class GoogleTiles {
```

Comment:
Class is just a translation of standard javascript for lon, lat translation to google tiles. Added reference to stackoverflow post.